grass7/imagery/i.sentinel/i.sentinel.coverage/i.sentinel.coverage.py

```diff
@@ -219,43 +223,73 @@ def main():
             producttype=producttype,
             start=options['start'],
             end=options['end'],
-            footprints=fps,
             flags='lb',
             quiet=True)
         if len(s_list) == 0:
             grass.fatal('No products found')
-        name_list = [x.split(' ')[1] for x in s_list]
+        name_list_tmp = [x.split(' ')[1] for x in s_list]
+    else:
+        name_list_tmp = options['names'].split(',')
+    name_list = []
+    fp_list = []
+    for name in name_list_tmp:
+        real_producttype, start_day, end_day = scenename_split(name)
+        if real_producttype != producttype:
+            grass.fatal(_
+                ("Producttype of {} not supported".format(real_producttype)))
```

Comment:
```suggestion
            grass.fatal(
                _("Producttype of {} not supported".format(real_producttype)))
```

(maybe no line-break in the gettext macro)